apps/services/auth/admin-api/src/app/v2/delegations/delegation-admin.controller.ts

```diff
@@ -91,6 +95,26 @@ export class DelegationAdminController {
     )
   }
 
+  @BypassAuth()
+  @UseGuards(ZendeskAuthGuard(ZENDESK_WEBHOOK_SECRET_GENERAL_MANDATE))
+  @Post(':zendeskId')
```

Comment:
Does Zendesk not support posting the id in the request body?

Reply:
it does, do we prefer that ?